playground/server/index.js

```diff
@@ -124,13 +124,13 @@ io.on('connection', socket => {
   });
 });
 
-app.get('/oauth_callback', (req, res) => {
+app.get('/auth-callback', (req, res) => {
   // Fulfill promise and make sure client closes the window
   resolveWebView(req.query);
   res.send('<script type="text/javascript">window.close()</script>');
 });
 app.use(express.static('./public'));
 
-server.listen(3000, () => {
-  console.log('Listening on *:3000');
+server.listen(8000, () => {
+  console.log('Listening on *:8000');
 });
```

Comment:
This will clash with Bloom if it's also running - not sure it's ideal to have both using the same port? 🤔

Reply:
That's exactly the point actually.
The idea is that you run one or the other, so we can use the same callback url.
This is not optimal but it's the easiest way to make sure the OAuth manager works with Bloom as the `getCallback()` method is called by the OAuth manager (else, we'd need to inject it somehow and that's a more complicated story)

Reply:
Ah okay, now I get it :)